tests/functional/restart/broadcast/flow.cylc

```diff
@@ -3,7 +3,6 @@
 [cylc]
     UTC mode = True
     [[events]]
-        abort if any task fails = True
```

Comment:
I can't see the matching test file change for this, is it one of those convoluted ones?

Reply:
It's `01-broadcast.t` - I just missed it.